AlfrescoApp/Model/Managers/RealmSyncManager.m

```diff
@@ -154,25 +418,149 @@ - (BOOL)isCurrentlySyncing
     return YES;
 }
 
-#pragma mark - Realm Utilities
-- (void)changeDefaultConfigurationForAccount:(UserAccount *)account
+- (NSString *)contentPathForNode:(AlfrescoDocument *)document
 {
-    RLMRealmConfiguration *config = [RLMRealmConfiguration defaultConfiguration];
+    RealmSyncNodeInfo *nodeInfo = [self.realmManager syncNodeInfoForObjectWithId:[self.syncHelper syncIdentifierForNode:document] inRealm:[RLMRealm defaultRealm]];
     
-    // Use the default directory, but replace the filename with the accountId
-    config.path = [self configPathForAccount:account];
+    //since this path was stored as a full path and not relative to the Documents folder, the following is necessary to get to the correct path for the node
+    NSString *newNodePath = nil;
+    if(nodeInfo)
+    {
+        NSString *storedPath = nodeInfo.syncContentPath;
+        NSString *relativePath = [self getRelativeSyncPath:storedPath];
+        NSString *syncDirectory = [[AlfrescoFileManager sharedManager] syncFolderPath];
+        newNodePath = [syncDirectory stringByAppendingPathComponent:relativePath];
+    }
     
-    // Set this as the configuration used for the default Realm
-    [RLMRealmConfiguration setDefaultConfiguration:config];
+    return newNodePath;
 }
 
-- (NSString *)configPathForAccount:(UserAccount *)account
+// this parses a path to get the relative path to the Sync folder
+- (NSString *) getRelativeSyncPath:(NSString *)oldPath
```

Comment:
Use “get” only for methods that return objects and values indirectly. You should use this form for methods only when multiple items need to be returned.
`- (void)getLineDash:(float *)pattern count:(int *)count phase:(float *)phase;`

[https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html](url)